pkg/virt-controller/services/template.go

```diff
@@ -1149,7 +1153,8 @@ func NewTemplateService(launcherImage string,
 	virtClient kubecli.KubevirtClient,
 	clusterConfig *virtconfig.ClusterConfig,
 	launcherSubGid int64,
-	exporterImage string) TemplateService {
+	exporterImage string,
+	opts ...TemplateServiceOption) TemplateService {
```

Comment:
nit: Please move the return type to its own separate line like [here](https://github.com/kubevirt/kubevirt/pull/10423/files#diff-e2b17b934fcd4fb2278da4c106432d96e03f936e34e2dd4854843501aa719ad8R1143).

Every time someone adds something the diff shows too much.

Reply:
Done